gateway/webrtc/client.js

```diff
@@ -140,22 +168,11 @@ function handleMessage(message) {
     console.log("Exiting handleMessage");
 };
 
-/*conn.onerror = function (err) {
-   console.log("Got error", err);
-};*/
-
-//alias for sending JSON encoded messages
-/*function send(message) {
-   //attach the other peer username to our messages
-   if (connectedUser) {
-      message.name = connectedUser;
-   }
-
-   conn.send(JSON.stringify(message));
-};*/
 
 function send(message) {
-
+    /*if (yourConn.iceGatheringState == "complete") {
```

Comment:
Is this comment needed, if so please specify what it does if you uncomment it.  Also are you missing a `;`